src/client/article.ts

```diff
@@ -327,21 +327,28 @@ function richLinks(): void {
 		.forEach((richLink) => {
 			const articleId = richLink.getAttribute('data-article-id');
 			if (articleId) {
-				void fetch(`${articleId}?richlink`).then((response) => {
-					const pillar = response.headers
-						.get('pillar')
-						?.toLowerCase();
-					const image = response.headers.get('image');
-
-					if (pillar) {
-						richLink.classList.add(`js-${pillar}`);
-					}
-
-					const placeholder = richLink.querySelector('.js-image');
-					if (placeholder && image) {
-						placeholder.innerHTML = `<img src="${image}" alt="related article"/>`;
-					}
-				});
+				const options = {
+					headers: {
+						Accept: 'text/html',
+					},
+				};
+				void fetch(`${articleId}?richlink`, options).then(
+					(response) => {
+						const pillar = response.headers
+							.get('pillar')
+							?.toLowerCase();
+						const image = response.headers.get('image');
+
+						if (pillar) {
+							richLink.classList.add(`js-${pillar}`);
+						}
+
+						const placeholder = richLink.querySelector('.js-image');
+						if (placeholder && image) {
+							placeholder.innerHTML = `<img src="${image}" alt="related article"/>`;
```

Comment:
Do you have an example of what the URL of an image will look like?

Reply:
This was a bit vague, sorry. There are a couple of reasons why I asked this.

1. I wanted to check that it was using the image resizer (`i.guim` rather than `media.guim`). I've done some digging and I think it does because we're pulling it from `mainMedia` on `Item`. 👍 

2. I wanted to check that we're making use of responsive images here. I'm assuming not because this is a straightforward `img[src]` - can we make use of `image-rendering` to include a responsive image?

Reply:
Yep it's using the same `src` as the main image. I did think about serving the `srcset` string to make it responsive but used `src` due to the small size of the image and the fixed with of the rich links.

Reply:
Ok, makes sense.

If we're not going with React here can we do something like this:

```ts
const img = document.createElement('img');
img.setAttribute('alt', 'Related article');
img.setAttribute('src', image);
placeholder.appendChild(img);
```

It's a bit more verbose but may be:

1) More performant, because there's no need to engage the HTML parser.
2) [More secure](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#Security_considerations)